README.md

```diff
@@ -11,13 +11,16 @@
 ```go
 import "go.linecorp.com/centraldogma"
 
-// create client with OAuth2 token
+// create a client with OAuth2 token
```

Comment:
It could be an OAuth2 token but it couldn't be depending on the configuration. And we are using SAML for authentication right now. So we should take `Oauth2` out. :-(

Reply:
Actually, by default, it's OAuth2 anyway since transport is nil. I think we could keep it like this to notify user.